src/ert/scheduler/openpbs_driver.py

```diff
@@ -56,6 +69,8 @@ def __init__(
         self._queue_name = queue_name
         self._memory_per_job = memory_per_job
         self._job_prefix = job_prefix
+        self._num_retries = 10
```

Comment:
Maybe hint in some way here that this is a peculiar feature when running certain command lines, and definitely not retry functionality for the job we are sending to the cluster.